dbt/include/athena/macros/materializations/models/incremental/incremental.sql

```diff
@@ -11,7 +12,10 @@
   {% set force_batch = config.get('force_batch', False) | as_bool -%}
   {% set target_relation = this.incorporate(type='table') %}
   {% set existing_relation = load_relation(this) %}
-  {% set tmp_relation = make_temp_relation(this) %}
+  {% set old_tmp_relation = adapter.get_relation(identifier=identifier ~ '__dbt_tmp',
+                                             schema=schema,
+                                             database=database) %}
+  {% set tmp_relation = make_temp_relation(target_relation) %}
```

Comment:
given the fact that you use `__dbt_tmp` for `old_tmp_relation`, shall we pass explicitely also `__dbt_tmp` as second argument to in `make_temp_relation `? e.g. `make_temp_relation(target_relatiom, '__dbt_tmp')`
just to have a crystal clear "table" suffix for tmp tables, and to align on what we do in the "table materialization", where we set table suffix esplicitely.

Reply:
@nicor88 Yeah for sure, then we aren't reliant on the default suffix.

Reply:
@nicor88 I have updated this now :)

Reply:
> Yeah for sure, then we aren't reliant on the default suffix.
Exactly, doing so the code is more robust to default upstream changes in dbt-core.
Thanks for adjusting the code.